CHANGELOG.md

```diff
@@ -4,18 +4,81 @@
 Add new items at the end of the relevant section under **Unreleased**.
 -->
 
-This project follows semantic versioning. While still in major version `0`,
-source-stability is only guaranteed within minor versions (e.g. between
-`0.0.3` and `0.0.4`). If you want to guard against potentially source-breaking
-package updates, you can specify your package dependency using
-`.upToNextMinor(from: "0.1.0")` as the requirement.
+This project follows semantic versioning.
 
 ## [Unreleased]
 
 *No new changes.*
 
 ---
 
+## [1.0.0] - 2021-09-07
+
+### Changes
+
+- Most sequence and collection types have been renamed, following a more
+  consistent naming structure:
+  - The `Lazy` prefix was dropped.
+  - Either a `Sequence` or `Collection` suffix was added depending on whether or
+    not the type is unconditionally a collection.
+  - The base name was derived from the name of the method that produces it,
+    including an argument label to disambiguate if necessary.
+  ```
+  Chain2              -> Chain2Sequence
+  ChunkedBy           -> ChunkedByCollection
+  ChunkedOn           -> ChunkedOnCollection
+  ChunkedByCount      -> ChunksOfCountCollection
+  Combinations        -> CombinationsSequence
+  Cycle               -> CycledSequence
+  FiniteCycle         -> CycledTimesCollection
+  Indexed             -> IndexedCollection
+  Intersperse         -> InterspersedSequence
+  LazySplitSequence   -> SplitSequence
+  LazySplitCollection -> SplitCollection
+  Permutations        -> PermutationsSequence
+  UniquePermutations  -> UniquePermutationsSequence
+  Product2            -> Product2Sequence
+  ExclusiveReductions -> ExclusiveReductionsSequence
+  InclusiveReductions -> InclusiveReductionsSequence
+  StrideSequence      -> StridingSequence
+  StrideCollection    -> StridingCollection
+  Uniqued             -> UniquedSequence
+  Windows             -> WindowsCollection
```

Comment:
IMO, "WindowedCollection" would be better aligned with the other type names (e.g. IndexedCollection, rather than IndexesCollection)

Reply:
I don't necessarily disagree, but we've arrived at these names by taking the base name of the method that produces them. In this case those are `windows(ofCount:)` and `indexed()`. The renaming of `ChunkedByCount` to `ChunksOfCountCollection` is another such example.

Reply:
🤷‍♂️ All of the other names seem fine, but this one just seems a bit off. Given that this is the 1.0 release, I just think it’s worth reconsidering is all. WindowsOfCountCollection, perhaps? Still a bit verbose for my liking, but better than WindowsCollection (which reads like something to do with Microsoft’s OS).

Reply:
@karwa Thanks for suggesting `WindowsOfCountCollection`! It indeed clears up any ambiguity and is still consistent with the other type names in this package, so that's the one we're going with.